src/store/actions.ts

```diff
@@ -116,7 +120,16 @@ export function setValgtTema(tema: string) {
     };
 }
 
-export function setKlageId(klageId: string) {
+export function setKlageId(klageId: string, tema: string = '*UNKNOWN*', ytelse: string = '*UNKNOWN*') {
+    sessionStorage.removeItem('nav.klage.klageId');
```

Comment:
Det er ikke nødvendig å fjerne før man setter en verdi; `sessionStorage.setItem('nav.klage.klageId', klageId);` vil oppdatere eller opprette verdien for `'nav.klage.klageId'`.

Gjelder også linjene under her :)

Reply:
Bra, da fikser jeg det. Mente jeg løste et problem med dette, men det var nok noe annet.